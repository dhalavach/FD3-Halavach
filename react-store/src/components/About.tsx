import React from 'react';

export default function About() {
  return (
    <div>
      About Us <br />
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Molestiae
      deleniti eligendi dolor, repellat beatae enim temporibus ea eveniet nihil
      ab assumenda illum quia aspernatur itaque sapiente hic ducimus incidunt
    </div>
  );
}
