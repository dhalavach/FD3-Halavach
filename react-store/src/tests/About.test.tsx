import '@testing-library/jest-dom';
import About from '../components/About';
import renderer from 'react-test-renderer';

test('component renders without throwing an error', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toBeTruthy();
});

test('component matches the snapshot', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
