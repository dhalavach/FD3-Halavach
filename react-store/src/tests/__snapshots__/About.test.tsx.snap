// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component matches the snapshot 1`] = `
<div>
  About Us 
  <br />
  Lorem, ipsum dolor sit amet consectetur adipisicing elit. Molestiae deleniti eligendi dolor, repellat beatae enim temporibus ea eveniet nihil ab assumenda illum quia aspernatur itaque sapiente hic ducimus incidunt
</div>
`;
